source/YearlyOverview/Yearly.js

```diff
@@ -0,0 +1,27 @@
+import { db } from '../Backend/FirebaseInit.js';
+import {
+    getUserID,
+    // setObjAtDBPath,
+    // getCurrentDate,
+} from '../../General/GlobalUtility.js';
```

Comment:
This is a good file that everyone on the team should know

Reply:
Is it alright if I remove this file? `Yearly.js` was meant to be a refactored version of YearlyJS.js

Reply:
file seems removed, it was used for testing but now it's functions are in `backendInit`